tools/metashrimps/DataPaper.xml

```diff
@@ -0,0 +1,78 @@
+<tool id="metashrimp_datapaper" name="Production of a draft of Data Paper" version="0.1.0+galaxy0" profile="22.05">
+    <description>Create a draft of Data Paper from metadata</description>
+    <requirements>
+       <requirement type="package" version="4.3.1">r-base</requirement>
+       <requirement type="package" version="1.1.2">r-dplyr</requirement>
+       <requirement type="package" version="1.3.5">r-xml2</requirement>
+       <requirement type="package" version="1.4.4">r-xslt</requirement>
+       <requirement type="package">r-mapview</requirement>
+       <requirement type="package" version="2.1.2">r-leaflet</requirement>
+       <requirement type="package">gmp</requirement>
+       <requirement type="package">phantomjs</requirement>
+       <requirement type="package" version="2.23">r-rmarkdown</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    Rscript
+        '$__tool_directory__/DataPaper.R'
+        '$__tool_directory__/template_emldown'
+        '$data'
+        '$editable'
+    ]]></command>
+    <inputs>
+        <param name="data" type="data" format="xml" label="Upload metadata" help="Insert only metadata using EML standard"/>
+        <param name="editable" type="select" label="Would you like to include a docx editable file instead of an HTML?" help="HTML files are better for visualisation but are more difficult to edit than a docx.">
+            <option selected="true" value="no">No</option>
+            <option value="yes">Yes</option>
+        </param>
+    </inputs>
+    <outputs>
+        <collection name="output1" type="list" label="Data Paper Draft">
```

Comment:
can you maybe use a more meaningful name here?

Reply:
Done